### DISCUSSION TITLE
Modifier le prix des classes du palier 1

### DISCUSSION THREAD

**Discussion by Thesmokingllama** [Crownicles/Crownicles#1171] [Idées acceptées / déjà implémentées]
étant investie dans la formation de nouvelles recrues sur le jeu, j'ai eu l'occasion de constater qu'une barrière importante empêche de nombreux joueurs de progresser au delà du niveau 10 : la mission n°7, changer de classe pour autre que recrue. Cette mission à la base anodine expose en réalité les moins expérimentés à un problème récurent dû à la répartition du prix des classes, ou plus précisément le prix de la classe lanceur de pierre, celle-ci étant la moins coûteuse du tier 1, est en effet très souvent choisie par défaut. Ma proposition serait donc d'augmenter le prix de cette classe afin qu'elle soit moins choisie et n'expose donc pas des joueurs à une mort d'autant plus certaine que leurs pv max tournent autour de 20 à 3O ❤️

**Comment by BastLast** [comment]
je note la suggestion, je propose de transformer cela en issue pour la prochaine maj

**Comment by BastLast** [comment]
les prix des classes de tier 1 seront divisés par 10 afin de permettre aux joueurs de les tester plus facilement :)

**Comment by BastLast** [comment]
les autres tiers, rien ne change pour le moment
